apps/passport-server/src/services/issuanceService.ts

```diff
@@ -1122,7 +1130,13 @@ export class IssuanceService {
               timestampConsumed: 0,
               isConsumed: false,
               isRevoked: false,
-              ticketCategory: TicketCategory.ZuConnect
+              ticketCategory: TicketCategory.ZuConnect,
+              imageUrl: urljoin(
```

Comment:
i wonder if there's any cache invalidation that needs to be accounted for here/ if we should solve that problem soon. fyi reason ive been deprioritizing it is anticipation of POD tickets. https://linear.app/0xparc-pcd/issue/0XP-424/make-ticket-caching-logic-robust-to-new-ticket-pcd-version

Reply:
A change to the PCD's fields will bust the cache. The only field that is ignored is `timestampSigned`.